Vagrantfile

```diff
@@ -41,7 +41,7 @@ arvados_cluster = {
         :ipaddress => "10.10.10.8",
         :memory => 512,
         :ansible_role => "shell"
-    }
+    },
```

Comment:
Nope, it seems to behave as Perl where the last comma is optional... I'll remove it anyway.